spec/decorators/contribution_decorator_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe ContributionDecorator do
+
+  describe '#user_short_name' do
+    subject {described_class.new(contribution).user_short_name}
```

Comment:
don't add the exercise step in the subject, avoid this, subject should be only build of a object

Reply:
I disagree here :stuck_out_tongue_closed_eyes: 
It is pretty common to make the subject points to the method you're actually exercising on

Reply:
it's common but not a good practice

Reply:
That's the reason we use the less descriptive version of subject for single-line exercises but for complex triggers, we usually give it a descriptive name. The point here is the format we usually expect for the file, I personally expect to see a `describe` block with a subject pointing to the method I'm dealing with.

```ruby
describe '#my_method' do
  subject { described_class.new.my_method }
  
  it { is_expected.to eq 10 }
end

describe 'GET /users' do
  subject(:make_request) { get '/users' }
  
  it 'expects something to happen' do
    make_request
    
    expect(response).to eq(...)
  end
end
```

But it's just my point, you can suggest any other format. :heart: :heart: :heart: